tests/test_operation.py

```diff
@@ -28,6 +29,17 @@
 op_classes_cv = [getattr(qml.ops, cls) for cls in qml.ops._cv__all__]
 op_classes_gaussian = [cls for cls in op_classes_cv if cls.supports_heisenberg]
 
+def PhaseShift(phi):
+    return np.exp(1j*phi/2)*Rotz(phi)
+
+def Rot(theta, phi, lam):
+    return Rotz(lam) @ Roty(phi) @ Rotz(theta)
```

Comment:
I think you can also import `PhaseShift` and `Rot` from default qubit!

Reply:
Good point, thanks!